vm/systemSmartContracts/delegation.go

```diff
@@ -1680,6 +1776,15 @@ func (d *delegation) unDelegate(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return vmcommon.UserError
 	}
 
+	entry := &vmcommon.LogEntry{
+		Identifier: []byte(args.Function),
+		Address:    args.CallerAddr,
+		Topics: [][]byte{
+			valueToUnDelegate.Bytes(), delegator.ActiveFund, globalFund.TotalActive.Bytes(),
```

Comment:
delegator.ActiveFund is only a key - it has no relevant information. You need the actual value as big.Int.bytes to return here

Reply:
fixed now use `remainedFund.Bytes()`

Reply:
refactored.